.changeset/light-plants-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"@caravan/bitcoin": major
```

Comment:
I think it's important to avoid major increments for small things if we're trying to encourage more developers to build using this library. I think we should collect a bunch of breaking changes and add them all at once.

As for the differing psbts, I don't believe that's a breaking change. The psbts have their own BIP defined required and optional values. You're allowed to not include an optional value (if that's what's being removed here) and consumers were supposed to have developed for that possibility. I think the only way this would have been breaking is if somewhere in the spec/docs of this library that value was guaranteed.